src/main/java/com/iexec/worker/docker/ComputationService.java

```diff
@@ -118,7 +125,7 @@ public boolean downloadApp(String chainTaskId, String appUri) {
         }
 
         String datasetFilename = FileHelper.getFilenameFromUri(datasetUri);
-        String datasetEnv = DATASET_FILENAME + "=" + datasetFilename;
+        String datasetEnv = IEXEC_DATASET_FILENAME_ENV_PROPERTY + "=" + datasetFilename;
```

Comment:
I think we need to support the same workflow (IEXEC_BOT_TASK_INDEX, etc ..) with TEE

Reply:
yes! Nice catch, I add it.

Reply:
done